app/assets/javascripts/short-form/ShortFormApplicationController.js.coffee

```diff
@@ -266,6 +266,9 @@ ShortFormApplicationController = (
   $scope.validateHouseholdEligibility = (match, callbackUrl) ->
     $scope.clearHouseholdErrorMessage()
     form = $scope.form.applicationForm
+    # skip the check if we're doing an incomeMatch and the applicant has vouchers
```

Comment:
@kaptron oh lol for some reason I was reading it as if the income passed validation, which is so silly coz i wrote this function. looks good to me!